### ISSUE TITLE
[a11y]: Focus is not visible on the home screen buttons when the background option is set to "Accent color"

### ISSUE THREAD

**Issue by TazmanianDI** [carbon-design-system/carbon-ai-chat#165]
![Image](https://github.com/user-attachments/assets/0730a2b0-86b5-435b-9322-07fb1d6e9ffc)
